### PR TITLE
cheange done by me

### PR DESCRIPTION
**Pull request by BhanuThakurani**

### CODE REVIEW

**Comment by ddobrigk** [comment]
Hi @BhanuThakurani , can you please edit the name of this PR to something more meaningful? Thank you!

File: DataFormats/Reconstruction/include/ReconstructionDataFormats/PID.h [hunk 1]
```diff
@@ -99,12 +100,14 @@ class PID
...
-  static constexpr ID HyperTriton = 13;
   static constexpr ID Hyperhydrog4 = 14;
   static constexpr ID XiMinus = 15;
   static constexpr ID OmegaMinus = 16;
+  // static constexpr ID FirstExt = PI0;
+  static constexpr ID Hyperhelium4 = 17;
+  static constexpr ID HyperTriton = 18;
   static constexpr ID FirstExt = PI0;
-  static constexpr ID LastExt = OmegaMinus;
+  static constexpr ID LastExt = Hyperhelium4;
```
Comment (shahor02) [inline DataFormats/Reconstruction/include/ReconstructionDataFormats/PID.h:110]: Why are you changing the IDs of existing particles? This is not backward-compatible. Also, with the `NIDsTot = 18` the specific IDs should span 0-17, while you reassign HyperTriton from 13 to 18.

File: Detectors/Vertexing/src/SVertexer.cxx [hunk 2]
```diff
@@ -12,41 +12,54 @@
...
+  "log_file.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pT4Hlambdalambda(
+  "log_pT4Hlambdalambda.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pT4He3Lambda(
+  "log_pT4He3Lambda.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pTpion(
+  "log_pTpion.txt", std::ios_base::out | std::ios_base::app);
+
+std::ofstream log_pT3body(
+  "log_pT3body.txt", std::ios_base::out | std::ios_base::app);
```
Comment (shahor02) [inline Detectors/Vertexing/src/SVertexer.cxx:46]: This is not used anywhere and we should not produce a stray log files in the standard mode

File: Detectors/Vertexing/src/SVertexer.cxx [hunk 3]
```diff
@@ -12,41 +12,54 @@
...
-  mNV0s = mNCascades = mN3Bodies = 0;
   updateTimeDependentParams(); // TODO RS: strictly speaking, one should do this only in case of the CCDB objects update
   mPVertices = recoData.getPrimaryVertices();
   buildT2V(recoData); // build track->vertex refs from vertex->track (if other workflow will need this, consider producing a message in the VertexTrackMatcher)
-  int ntrP = mTracksPool[POS].size(), ntrN = mTracksPool[NEG].size();
-  if (mStrTracker) {
-    mStrTracker->loadData(recoData);
-    mStrTracker->prepareITStracks();
-  }
+  int ntrP = mTracksPool[POS].size(), ntrN = mTracksPool[NEG].size(), iThread = 0;
```
Comment (shahor02) [inline Detectors/Vertexing/src/SVertexer.cxx:54]: Too many modifications below in quite a subtle code and not clear why they are needed, please clarify.

**Review by shahor02 — CHANGES_REQUESTED** [review]

**Comment by sawenzel** [comment]
There is no reaction to the comments in the this PR. I will close it for now. Please rework it and reopen when ready (good commit message and reasonable PR title).

**Comment by ddobrigk** [comment]
I think this PR was opened by accident and indeed fine to close. I will discuss with the authors in more detail how to proceed. Thanks!

**Pull request closed without merging**